File: src/main/java/com/example/ajaxonboardingproject/model/CinemaHall.java
package com.example.ajaxonboardingproject.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Entity
@Data
@Table(name = "cinema_halls")
public class CinemaHall {
    @Id
    @EqualsAndHashCode.Exclude
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private int capacity;
    private String description;
}

File: src/main/java/com/example/ajaxonboardingproject/model/CinemaHall.kt
package com.example.ajaxonboardingproject.model

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document("cinemaHalls")
data class CinemaHall(
    val capacity: Int,
    val description: String,
) {
    @Id
    lateinit var id: String
}
